smf/ntp/manifest/manifest.xml

```diff
@@ -67,29 +67,24 @@
 
         <exec_method type="method"
                      name="refresh"
-		     exec="/var/svc/method/svc-site-ntp %m %{restarter/contract}"
+                     exec="/var/svc/method/svc-site-ntp %m %{restarter/contract}"
                      timeout_seconds="60">
         </exec_method>
 
         <exec_method type="method"
                      name="stop"
-		     exec="/var/svc/method/svc-site-ntp %m %{restarter/contract}"
+                     exec="/var/svc/method/svc-site-ntp %m %{restarter/contract}"
                      timeout_seconds="60" />
 
-        <property_group name="config"
-                        type="application">
-            <propval name="file"
-                     type="astring"
-                     value="/etc/inet/chrony.conf" />
-	    <propval name="boundary"
-                     type="boolean"
-		    value="false" />
-            <propval name="server"
-                     type="astring"
-                     value="" />
-            <propval name="allow"
-                     type="astring"
-                     value="" />
+        <property_group name="config" type="application">
+            <propval name='all_links' type='astring' value='unknown' />
+            <propval name='datalink' type='astring' value='unknown' />
+            <propval name='gateway' type='astring' value='unknown' />
+            <propval name='listen_addr' type='astring' value='unknown' />
```

Comment:
These could be `type='net_address_v6'` (assuming they are always IPv6 addresses)